src/test/java/com/google/neighborgood/IntegrationTest.java

```diff
@@ -0,0 +1,1065 @@
+              }
+            })
+        .click();
+
+    // clicks on confirm help button
+    wait.until(
+            new Function<WebDriver, WebElement>() {
+              public WebElement apply(WebDriver driver) {
+                return driver.findElement(By.xpath(taskXPath + "/div[1]/a"));
+              }
+            })
+        .click();
+    driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+
+    // Updates recentTask
+    recentTask.put("helper", USER_NICKNAME_HELPER);
+    recentTask.put("status", "IN PROGRESS");
+  }
+
+  /**
+   * Verifies that after claiming a task/offering help with a task, that it displays correctly in
+   * the user page's offer help table
+   */
+  private void verifyOfferHelpTask() {
+    // Location of most recent task in offer help table in userpage
+    String offerHelpRowXPath = "//tbody[@id='offer-help-body']/tr[1]";
+
+    String taskOverview =
+        wait.until(
+                new Function<WebDriver, WebElement>() {
+                  public WebElement apply(WebDriver driver) {
+                    return driver.findElement(By.xpath(offerHelpRowXPath + "/td[1]"));
+                  }
+                })
+            .getText();
+    assertEquals(recentTask.get("overview"), taskOverview);
+
+    String taskStatus =
+        wait.until(
+                new Function<WebDriver, WebElement>() {
+                  public WebElement apply(WebDriver driver) {
+                    return driver.findElement(By.xpath(offerHelpRowXPath + "/td[2]"));
+                  }
+                })
+            .getText();
+    assertEquals(recentTask.get("status"), taskStatus);
+
+    String taskNeighbor =
+        wait.until(
+                new Function<WebDriver, WebElement>() {
+                  public WebElement apply(WebDriver driver) {
+                    return driver.findElement(By.xpath(offerHelpRowXPath + "/td[3]"));
+                  }
+                })
+            .getText();
+    assertEquals(recentTask.get("nickname"), taskNeighbor);
+
+    verifyTaskDetails(offerHelpRowXPath + "/td[1]");
+  }
+
+  /** Logs out user - takes logout link id as a parameter */
+  private void logOut(String logoutId) {
+    driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+    // clicks on logout button
+    wait.until(
+            new Function<WebDriver, WebElement>() {
+              public WebElement apply(WebDriver driver) {
+                return driver.findElement(By.id(logoutId));
+              }
+            })
+        .click();
+    driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+  }
+
+  /**
+   * Updates the stored instance of recentTask with the most recent open task's contents displayed
+   * in the homepage
+   */
+  private void updateRecentTask() {
+    recentTask.clear();
+
+    // Most recent task location in homepage
+    String taskXPath = "//div[@id='tasks-list']/div[1]/div[2]";
+
+    String taskNickname =
```

Comment:
Same suggestion as the suggestion for `verifyNewTaskHomepage()`: using a loop here and treat the situation when i == 2 differently

Reply:
See above :)